### PR TITLE
[#P-235] Handle ProblemJson from fn-eucovidcert and return it

### PR DESCRIPTION
**Pull request by gquadrati**
#### List of Changes

Add ProblemJson object to responses with status code 400 and 500. 

**NOTE: Wait PR https://github.com/pagopa/io-functions-eucovidcerts/pull/26 to be merged**

#### Motivation and Context

In order ti properly generate client handler in io-backend, add missing return ProblemJson 

#### How Has This Been Tested?

io-mock

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/services/eucovidcertService.ts [hunk 1]
```diff
@@ -64,7 +65,7 @@ export default class EUCovidCertService {
           case 403:
             return ResponseErrorNotFound("Not Found", "Certificate not found");
           case 500:
-            return ResponseErrorInternal("");
+            return ResponseErrorInternal(readableProblem(response.value));
           case 504:
             return ResponseErrorInternal("");
```
Comment (BurnedMarshal) [inline src/services/eucovidcertService.ts:70]: Doesn't is any error message detail for this response.

Comment (gquadrati) [reply src/services/eucovidcertService.ts:70]: there is now that I merged the fn-eucovidcert PR I mentioned ( pagopa/io-functions-eucovidcerts#26 )

**Review by fabriziopapi — APPROVED** [review]
with eucovidcerts merged, it works fine

File: src/services/eucovidcertService.ts (hunk 1 above)
Comment (gquadrati) [reply src/services/eucovidcertService.ts]: Done in [c1258c4](https://github.com/pagopa/io-backend/pull/808/commits/c1258c46d0f4d17961d69482e74ed7547a9d5c85)

**Review by balanza — APPROVED** [review]

**Pull request merged**